torch/csrc/jit/serialization/export.h

```diff
@@ -21,7 +25,13 @@ namespace jit {
 // file contents being the raw tensor data.
 using RawDataExportMap = std::unordered_map<std::string, at::Tensor>;
 
-TORCH_API std::tuple<std::string, RawDataExportMap> export_onnx(
+using SymbolDimMap = std::map<c10::ShapeSymbol, std::string>;
```

Comment:
I second the choice of using map instead unordered_map to this run-to-run deterministic. This map is small enough where this should not have any significant perf impact.